### DISCUSSION TITLE
Is RabbitMQ 4.0 supported by the operator?

### DISCUSSION THREAD

**Discussion by bo0ts** [rabbitmq/cluster-operator#1752] [Q&A] [answered]
Since the operator by default deploys version 3.13.7 (which has already reached end of community support) I was wondering if 4.0 releases of RabbitMQ are already supported by the operator? 

Also, are upgrades from 3.13.7 supported by the operator? Are changes to the manifests required?

Maybe a clarification for that in the docs would be helpful.

**Comment by mkuratczyk** [comment, marked as answer]
https://github.com/rabbitmq/cluster-operator/discussions/1746
